insertBaseData-psql.sql

```diff
@@ -236,12 +239,22 @@ INSERT INTO "locationRelationships" ("childLocationUuid", "parentLocationUuid")
   ('22b0137c-4d89-43eb-ac95-a9f68aba884f', (SELECT uuid FROM locations WHERE type = 'PAC' AND name = 'Afghanistan')),
   ('60f4084f-3304-4cd5-89df-353edef07d18', (SELECT uuid FROM locations WHERE type = 'PAC' AND name = 'Afghanistan')),
   ('c136bf89-cc24-43a5-8f51-0f41dfc9ab77', (SELECT uuid FROM locations WHERE type = 'PAC' AND name = 'Afghanistan')),
-  ('b0979678-0ed0-4b42-9b26-9976fcfa1b81', (SELECT uuid FROM locations WHERE type = 'PAC' AND name = 'Afghanistan'));
+  ('b0979678-0ed0-4b42-9b26-9976fcfa1b81', (SELECT uuid FROM locations WHERE type = 'PAC' AND name = 'Afghanistan')),
+  ('9f83fe70-e9f5-4e92-ae48-5c4fd7076f46', '92fbce65-8fa9-403b-a5a2-f0c3189e4f9a'),
+  ('178dfbba-f15a-400b-9135-6ff800246be0', '9f83fe70-e9f5-4e92-ae48-5c4fd7076f46');
 
 UPDATE locations
 SET "customFields"='{"invisibleCustomFields":["formCustomFields.textareaFieldName","formCustomFields.numberFieldName"],"arrayFieldName":[],"nlt_dt":null,"nlt":null,"colourOptions":"","inputFieldName":"consectetur adipisici elit","multipleButtons":[]}'
 WHERE name='MoI Coffee Shop';
 
+UPDATE locations
+SET "customFields"='{"townId":39,"townSerbian":"Decani","townAlbanian":"Decan","municipalitySerbian":"Decani","municipalityAlbanian":"Decan","mgrs":"34TDN41700980"}'
```

Comment:
Contents of `customFields` column should be:
```
 {"invisibleCustomFields":[],"townId":"39","townSerbian":"Decani","townAlbanian":"Decan","municipalitySerbian":"Decani","municipalityAlbanian":"Decan","mgrs":"34TDN41700980"}
```
To be able to view them in ANET, consider defining them in the dictionary.